common/predictive-text/build.sh

```diff
@@ -180,7 +199,8 @@ if (( fetch_deps )); then
   npm install --no-optional
 fi
 
```

Comment:
We may want an explicit common build step called something like `build-wrapped-worker` code. The results of this is later consumed by `build-browser` and `build-headless`.

```suggestion

build-wrapped-worker || fail "LMLayer worker compilation failed"
```

Reply:
I haven't gotten to this point yet, but I think I've covered the others now.

I'll work on that one in the morning, I suppose.  Shouldn't take long when I'm fresh.